tests/example-config.yml

```diff
@@ -0,0 +1,18 @@
+# This is an example test config file to be used with the run-all.go file.
+keda:
+  installKeda: true # e2e tests will setup KEDA for you
+  installKafka: true # e2e tests will setup Kafka for you
+  imageRegistry: "" # use KEDA images from registry
+  imageRepo: "" # use KEDA images from repo
+# The e2e launcher will only run the cpu scaler test.
+testCategories:
+  internals:
+    mode: exclude
+  scalers:
+    mode: include
+    tests:
+      - cpu
```

Comment:
Based on the code, I'd say that this has to be the name of the tests file and not just the scaler (it's not a problem for me, but just to clarify this in examples if it's the case)

Reply:
I address this in above, it's because I'm matching directory names not the test files themselves. If we don't like that, then let me know :)